### ISSUE TITLE
Welcome Tour: Tour stays in front of the menu on smaller screens

### ISSUE THREAD

**Issue by StefanNieuwenhuis** [Automattic/wp-calypso#55610]
### Steps to reproduce the behavior

1. Open a browser and switch to a mobile viewport
2. Go to the Editor.
3. Open/Activate the Welcome Tour
4. Click on a menu item
5. The menu opens **behind** the Tour.

#### What I expected to happen
When the Welcome Tour is open, and you click on a menu item, the Tour should be minimized.

#### What actually happened
When the Welcome Tour is open, and you click on a menu item, the tour stays open.

<img width="389" alt="Screenshot 2021-08-18 at 17 04 00" src="https://user-images.githubusercontent.com/12104589/130095469-4dbcb572-cbfb-4923-a88e-5b2ea7eead11.png">

**Comment by razvanpapadopol** [comment]
Hey, @poligilad-auto, could you have a look at the mobile experience of the Welcome tour?
Should we keep it as a floating panel with the rest of the UI visible (as it is on desktop) or make it full-screen with the possibility to see what's behind only if you minimize it?

In case we keep it as it is, do you think the following is a good solution to avoid overlapping?
> When the Welcome Tour is open, and you click on a menu item, the Tour should be minimized.

**Comment by poligilad-auto** [comment]
Hey @razvanpapadopol, I think we can keep the same behaviour like in desktop view.

> When the Welcome Tour is open, and you click on a menu item, the Tour should be minimized.


I agree with this solution @StefanNieuwenhuis

**Comment by dpasque** [comment]
Bringing priority down to Normal due to availability of easy work around (in closing or dismissing the tour) and that the tour is not present every time you go to editor.
